public/translations/pt-BR.json

```diff
@@ -194,8 +197,8 @@
       "twelve_words": "Suas 12 palavras",
       "private_key": "Sua chave privada",
       "recommendation": "Recomendamos que você salve essas informações clicando no botão abaixo. Se você perder, não poderá fazer login novamente na sua conta.",
-      "download": "Salvar chave privada e 12 palavras",
-      "instructions": "Você salvará a chave privada e as 12 palavras em seu computador no formato PDF."
+      "download": "Criar a minha conta",
+      "instructions": "Copie e cole as 12 palavras abaixo em um local seguro. Você tem apenas essa chance de fazê-lo; se você perdeu essas palavras, nunca mais poderá acessar sua conta."
```

Comment:
maybe a more "common" way would be 

> Copie e cole as 12 palavras abaixo em um local seguro. Essa é a sua única chance de fazê-lo; em caso de perda das palavras, você não poderá mais recuperar ou acessar sua conta 

Any thoughts @heltonlr @RodrigoLeiteF

Reply:
Works to me